# test: {"ret": 1}
def func():
	raise Exception('error')

def main():
	func()
